ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1165,7 +1170,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
+                                <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                                <Button Click="HUDAndVisuals_Reset" Width="22" Height="22" ToolTip="{DynamicResource ServerSettings_ResetSectionTooltip}" Margin="20,0,0,0">
```

Comment:
I have tried and tried to get the commands working for these buttons but without any luck. It does not seem to bind correctly and the buttons do nothing.

Any chance you could implement one of the buttons with an example? I am so confused!